extensions/data-plane/data-plane-framework/src/test/java/org/eclipse/dataspaceconnector/dataplane/framework/DataPlaneFrameworkExtensionTest.java

```diff
@@ -0,0 +1,31 @@
+package org.eclipse.dataspaceconnector.dataplane.framework;
+
+import org.eclipse.dataspaceconnector.dataplane.framework.manager.DataPlaneManagerImpl;
+import org.eclipse.dataspaceconnector.dataplane.framework.pipeline.PipelineServiceImpl;
+import org.eclipse.dataspaceconnector.dataplane.spi.manager.DataPlaneManager;
+import org.eclipse.dataspaceconnector.dataplane.spi.pipeline.PipelineService;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+import org.junit.jupiter.api.Test;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+class DataPlaneFrameworkExtensionTest {
```

Comment:
Isn't this test just duplicating the implementation? Consider removing.

Reply:
Right, indeed the lack of testability is what drove #134. We can adapt then.

For now it has a little bit of extra value - that no exceptions are thrown.